src/frontend/komponenter/Behandling/Aktivitet/Aktivitet.tsx

```diff
@@ -43,18 +43,14 @@ const Aktivitet: FC<Props> = ({ behandlingId }) => {
     }, [postOvergangsstønadSuksess]);
 
     const ferdigVurdert = (behandlingId: string): any => {
-        const postOvergangsstønad = (): Promise<Ressurs<string>> => {
-            return axiosRequest<any, any>({
-                method: 'POST',
-                url: `/familie-ef-sak/api/vurdering/${behandlingId}/overgangsstonad/fullfor`,
-            });
+        const postOvergangsstønad = () => {
+            history.push(`/behandling/${behandlingId}/vedtak-og-beregning`);
         };
-
+        hentBehandling.rerun();
```

Comment:
Notér i refaktorer-boka; Slett all kode som handler om `postOvergangsstønadSuksess` / `feilmelding` / `ferdigVurdert`

Hvis vi skal ha knapp så skal den bare gjøre `history.push` - men tror vi kan fjerne knappen også :)

Reply:
Notert :)